### PR TITLE
Features/8618 - Keycloak configuration doc

### PR DESCRIPTION
**Pull request by bradhead**
## Status
**Ready**

## Fixes or Implements [AB#8618](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/8618)
- [ ] Enhancement
- [ ] Bug
- [x] Documentation

## Description:
Created README.md to document Keycloak configurations. 

## Testing
- [ ] Unit Tests Updated
- [ ] Functional Tests Updated
- [X] Not Required

### Steps to Reproduce:
Not a bug

### UI Changes
NO

### Browsers Tested
- [x] Chrome
- [ ] Safari
- [ ] Edge
- [ ] Firefox

Provide an explanation for any test(s) not completed.

## Notes/Additional Comments
None

### CODE REVIEW

File: Testing/load/timeline_load_k6.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ let users = [
   { username: "loadtest_10", password: passwd, hdid: null, token: null, refresh: null, expires: null },
   { username: "loadtest_11", password: passwd, hdid: null, token: null, refresh: null, expires: null },
   { username: "loadtest_12", password: passwd, hdid: null, token: null, refresh: null, expires: null },
-//  { username: "loadtest_13", password: passwd, hdid: null, token: null, refresh: null, expires: null },
```
Comment (sslaws) [inline Testing/load/timeline_load_k6.js:45]: NIT:  Adding additional code or other changes should be documented in the PR.

File: Tools/Dev/Keycloak/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# HealthGateway KeyCloak Configuration Guide
+
+The HealthGateway uses Keycloak, an OAuth2 compliant authorization server to control access to its protected resources. It connects the user via OIDC to external Identity Providers, in particular the BC Services Card identity and authentication service.   For administrators Keycloak also uses government employee's Active Directory services. 
+
+## Environments
+
+The HealthGateway project has three realms assigned to it for Dev, Test and production. Each environment is configured separately:
```
Comment (sslaws) [inline Tools/Dev/Keycloak/README.md:7]: NIT:  Dev and Test are capitalized while Production is not.

File: Testing/load/timeline_load_k6.js (hunk 1 above)
Comment (bradhead) [reply Testing/load/timeline_load_k6.js]: I am not sure how this go updated. it wasn't in my change log

Comment (bradhead) [reply Testing/load/timeline_load_k6.js]: not sure how this change floated in.. it was the same before.

File: Tools/Dev/Keycloak/README.md (hunk 2 above)
Comment (bradhead) [reply Tools/Dev/Keycloak/README.md:7]: fixed

**Pull request merged**